pkg/reconciler/apis/apibinding/apibinding_reconcile.go

```diff
@@ -519,3 +545,30 @@ func apiExportLatestResourceSchemasChanged(apiBinding *apisv1alpha1.APIBinding,
 
 	return !exportedSchemaUIDs.Equal(boundSchemaUIDs)
 }
+
+func propagateLabelAndAnnotations(source, target metav1.Object) {
```

Comment:
would rather expect a positive list. Why should we copy other annotation?

Reply:
I wasn't sure, but I could see values in copying the set of annotations and labels, for metering for instance, or simply to convey the annotations that the API providers generally set. For example the Gateway API set a number of annotations with the `gateway.networking.k8s.io` prefix.

Reply:
But nobody will see these CRDs. And there is the risk of side-effect, even security related.

Reply:
Make sense, let me rework it quickly.